src/sql/migrations.rs

```diff
@@ -1169,6 +1169,19 @@ CREATE INDEX msgs_status_updates_index2 ON msgs_status_updates (uid);
         .await?;
     }
 
+    inc_and_check(&mut migration_version, 130)?;
+    if dbversion < migration_version {
+        // Speed up `Message.get_saved_msg_id()`.
+        //
+        // Otherwise checking if the message is "bookmarked"
+        // is expensive and messages in the chats load slowly.
+        sql.execute_migration(
+            "CREATE INDEX msgs_index_starred ON msgs (starred);",
```

Comment:
hm, that time i also thought about an index, and some lines above, at line 120, there is already `CREATE INDEX msgs_index5 ON msgs (starred)`.

so, not sure why that is not used. may it be that it was deleted on some machines? maybe manually by some dev?

Reply:
sounds all a bit weird :)

if in doubt, maybe drop the old index in the same migration where we add the new one